src/webserver/Server.h

```diff
@@ -130,11 +125,11 @@ class Server : public std::enable_shared_from_this<Server<PlainSession, SslSessi
     using std::enable_shared_from_this<Server<PlainSession, SslSession, Handler>>::shared_from_this;
 
     clio::Logger log_{"WebServer"};
-    std::reference_wrapper<boost::asio::io_context> const ioc_;
-    std::optional<std::reference_wrapper<boost::asio::ssl::context>> const ctx_;
-    util::TagDecoratorFactory const tagFactory_;
-    std::reference_wrapper<clio::DOSGuard> const dosGuard_;
-    std::shared_ptr<Handler> const handler_;
+    std::reference_wrapper<boost::asio::io_context> ioc_;
```

Comment:
why remove these const?

Reply:
const and reference members are generally a code smell. in this case it may not matter for copy-assignability but i still prefer to follow core guidelines whenever we can. one day we may integrate clang-tidy and then this would come up as a failed check anyways: https://clang.llvm.org/extra/clang-tidy/checks/cppcoreguidelines/avoid-const-or-ref-data-members.html